themes/bootstrap3/js/lightbox.js

```diff
@@ -523,6 +523,13 @@ VuFind.register('lightbox', function Lightbox() {
     });
     _modal.on("shown.bs.modal", function lightboxShown() {
       bindFocus();
+
+      // Disable bootstrap-accessibility.js "enforceFocus" events.
+      // retainFocus() above handles it better.
+      // This is moot once that library (and bootstrap3) are retired.
+      var focEls = _modal.find(":tabbable");
+      var lastEl = $(focEls[focEls.length-1]);
+      $(lastEl).off('keydown.bs.modal');
```

Comment:
The ugly variable names are just echoing what's in bootstrap-accessibility, for easier reference.